src/components/container/sticky-bar/sticky-bar.stories.ts

```diff
@@ -0,0 +1,155 @@
+import type { InputType } from '@storybook/types';
+import type { ArgTypes, Args, Meta, StoryObj } from '@storybook/web-components';
+import { type TemplateResult, html } from 'lit';
+
+import { sbbSpread } from '../../core/dom';
+
+import '../../button';
+import '../../title';
+import readme from './readme.md?raw';
+import '../container';
+import './sticky-bar';
+
+const expanded: InputType = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const color: InputType = {
+  control: {
+    type: 'select',
+  },
+  options: ['transparent', 'white', 'milk', 'midnight'],
+};
+
+const defaultArgTypes: ArgTypes = {
+  expanded,
+  color,
+};
+
+const defaultArgs: Args = {
+  expanded: false,
+  color: color.options[0],
+};
+
+const containerContent = (title: string, color: string): TemplateResult => html`
+  <div style="overflow: auto;">
+    <sbb-title level="4" ?negative=${color === 'midnight'}>${title}</sbb-title>
+    <p class="sbb-text-s">The container component will give its content the correct spacing.</p>
+    <p class="sbb-text-s">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+      voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+    </p>
+  </div>
+  <sbb-button
+    variant="secondary"
+    ?negative=${color === 'midnight'}
+    style="margin-block-end: 0.75rem;"
+    >See more</sbb-button
+  >
+`;
+
+const Template = (): TemplateResult =>
+  html` <sbb-sticky-bar>
+    <sbb-button variant="secondary">Example</sbb-button>
+  </sbb-sticky-bar>`;
+
+const DefaultTemplate = ({ color, ...args }: Args): TemplateResult => html`
+  <sbb-container ${sbbSpread(args)} color=${color}>
+    ${containerContent('Example title', color)} ${containerContent('Another one', color)}
+    ${containerContent('And another one', color)} ${containerContent('And a last one', color)}
+
+    <sbb-sticky-bar>
+      <sbb-button style="margin-inline-start: auto">Continue</sbb-button>
+    </sbb-sticky-bar>
+  </sbb-container>
+`;
+
+const ShortTemplate = ({ color, ...args }: Args): TemplateResult => html`
+  <sbb-container ${sbbSpread(args)} color=${color}>
+    ${containerContent('Example title', color)}
+
+    <sbb-sticky-bar>
+      <sbb-button style="margin-inline-start: auto">Continue</sbb-button>
```

Comment:
Maybe nest in an action group? What would be the best example here? @mcilurzo

Reply:
an action-group would be nice.